Makefile

```diff
@@ -100,16 +97,15 @@ link: uninstall
 	node bin/npm-cli.js link -f --ignore-scripts
 
 prune: deps
-	node bin/npm-cli.js prune --production --no-save --no-audit
-	@[[ "$(shell git status -s)" != "" ]] && echo "ERR: found unpruned files" && exit 1 || echo "git status is clean"
+	node bin/npm-cli.js prune --production --no-save --no-audit --no-fund
+	node scripts/git-dirty.js
 
-publish: gitclean ls-ok link test smoke-tests docs prune
-	@git push origin :v$(shell node bin/npm-cli.js --no-timing -v) 2>&1 || true
```

Comment:
Our release process always includes us creating the tag before running this step, so this was unnecessary. It wasn't erroring but would show the message that the tag already existed.